.github/dependabot.yml

```
@@ -31,8 +29,8 @@ updates:
     commit-message:
       prefix: "docs"
       include: "scope"
-    rebase-strategy: disabled
-    versioning-strategy: increase-if-necessary
     groups:
-      dev-deps:
-        dependency-type: "development"
+      mkdocs:
+        patterns:
+          - "mkdocs*"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)